### DISCUSSION TITLE
Contribute project

### DISCUSSION THREAD

**Discussion by parteekcoder** [ControlCore-Project/concore#65] [General]
@pradeeban I have mailed you my proposal and you have also reviewed that,
How it sounds to you

I didn't receive reviews from other mentors, is I am lacking somewhere

**Comment by pradeeban** [comment]
@parteekcoder, your proposal now looks good. Mentors don't comment unless they have something to say. (So, you won't hear from anyone along the lines of, "Looks good" or "I have no comments"). No need to send reminders to them.

Feel free to continue to edit, and make sure to submit your final proposal on time to the GSoC site.
